docs/genindex.html

```diff
@@ -281,172 +223,34 @@ <h2 id="D">D</h2>
   </dt>
 
       
-  <dt><a href="dynamic_rest.html#dynamic_rest.fields.DynamicComputedField">DynamicComputedField (class in dynamic_rest.fields)</a>
```

Comment:
What happened to all of the `dynamic_rest.fields` documentation? Surprised to see it redlined when there are no actual fields changes in this PR